### PR TITLE
Fix xcm-builder mock

### PR DESCRIPTION
**Pull request by ggwpez**
(preparation for the monorepo)

The CI fails here when the runtime-benchmarks feature is enabled in the workspace.

### CODE REVIEW

**Review by acatangiu — APPROVED** [review]

**Comment by ggwpez** [comment]
bot merge

File: xcm/xcm-builder/Cargo.toml [hunk 1]
```diff
@@ -42,6 +42,10 @@ runtime-benchmarks = [
 	"frame-support/runtime-benchmarks",
 	"frame-system/runtime-benchmarks",
 	"xcm-executor/runtime-benchmarks",
+	"pallet-assets/runtime-benchmarks",
+	"pallet-balances/runtime-benchmarks",
+	"pallet-salary/runtime-benchmarks",
+	"pallet-xcm/runtime-benchmarks",
```
Comment (ggwpez) [inline xcm/xcm-builder/Cargo.toml:48]:
```suggestion
```

**Comment by ggwpez** [comment]
bot merge

**Comment by ggwpez** [comment]
bot cancel

**Comment by mordamax** [comment]
bot merge cancel

**Review by kianenigma — APPROVED** [review]

**Pull request merged**